[2023-09-22T16:52:17.314+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:52:17.509+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name__1 scheduled__2021-11-16T00:00:00+00:00 [queued]>
[2023-09-22T16:52:17.538+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name__1 scheduled__2021-11-16T00:00:00+00:00 [queued]>
[2023-09-22T16:52:17.541+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 6
[2023-09-22T16:52:17.586+0000] {taskinstance.py:1350} INFO - Executing <Task(_PythonDecoratedOperator): get_name__1> on 2021-11-16 00:00:00+00:00
[2023-09-22T16:52:17.607+0000] {standard_task_runner.py:57} INFO - Started process 495 to run task
[2023-09-22T16:52:17.624+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_name__1', 'scheduled__2021-11-16T00:00:00+00:00', '--job-id', '110', '--raw', '--subdir', 'DAGS_FOLDER/test2.py', '--cfg-path', '/tmp/tmpmolhlguz']
[2023-09-22T16:52:17.648+0000] {standard_task_runner.py:85} INFO - Job 110: Subtask get_name__1
[2023-09-22T16:52:17.691+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:188 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:52:17.773+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_name__1 scheduled__2021-11-16T00:00:00+00:00 [running]> on host 75cee4990e54
[2023-09-22T16:52:17.846+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:52:20.265+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_name__1' AIRFLOW_CTX_EXECUTION_DATE='2021-11-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-16T00:00:00+00:00'
[2023-09-22T16:52:20.269+0000] {python.py:183} INFO - Done. Returned value was: {'first_name': 'Jerry', 'last_name': 'Fridman'}
[2023-09-22T16:52:20.407+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_name__1, execution_date=20211116T000000, start_date=20230922T165217, end_date=20230922T165220
[2023-09-22T16:52:20.529+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-09-22T16:52:20.759+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
